ckanext/data_qld/resource_visibility/validators.py

```diff
@@ -1,54 +1,40 @@
 # encoding: utf-8
 
-from ckantoolkit import _, missing, StopOnError
+import ckan.plugins.toolkit as tk
 
-from ckanext.data_qld.resource_visibility import helpers as resource_visibility_helpers
-from ckanext.data_qld import helpers as data_qld_helpers
+ValidationError = tk.ValidationError
+_ = tk._
 
 
-def resource_visibility(key, data, errors, context):
+def resource_visibility(value):
```

Comment:
These three functions are essentially identical, just with different field names in the error message. They could be refactored.